<launch>

    <!-- Run the map server -->
    <arg name="map_file_1" default="$(find robotcraft_maze_pro_real)/world/gen_map_l.yaml"/>
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file_1)">
    <remap from="/map" to="/map_1"/>
    </node>

    <arg name="map_file" default="$(find robotcraft_maze_pro_real)/world/gen_map.yaml"/>
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)">
    </node>


</launch>
